solutions/cf-1418C.mdx

```diff
@@ -84,12 +84,11 @@ import java.io.*;
 import java.util.Arrays;
 
 public class MortalKombatTower {
-	public static final int INTMAX = 1000000000;
 	public static void main(String[] args) throws IOException {
 		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
 		int t = Integer.parseInt(br.readLine());
 
-		for (int i = 0; i < t; i++) {
+		for (int testcase = 0; testcase < t; testcase++) {
```

Comment:
Uh please do t not test_case

Reply:
well t is given in the problem statement as a variable. How should I name this variable then if I use t as ite var?

Reply:
`test_case_num` as the actual variable and `t` for the iteration

Reply:
I'd prefer `while (t-->0)`, without spaces as I think the `-->` looks like an arrow, but `for (; t > 0; t--)` is probably the most clear.

Reply:
nah fam